podium-gateway/src/main/webapp/app/shared/request/action-bars/request-action-toolbar/request-action-toolbar.component.html

```diff
@@ -9,40 +9,134 @@
   -->
 
 <div class="request-action-toolbar" *ngIf="request">
-    <!-- Draft / Submit -->
-    <div class="action-container" *ngIf="isStatus(requestStatus.Draft)">
-        <button type="button" class="btn btn-danger" (click)="resetForm()">Clear</button>
-        <div class="float-xs-right">
-            <button type="button" class="btn btn-primary" (click)="saveDraft()">Save draft</button>
-            <button type="submit" class="btn btn-success" [disabled]="!form.form.valid"
-                    (click)="submitDraft()">Submit</button>
-        </div>
-    </div>
-
-    <!-- Validate in request details -->
-    <div class="action-container"  *ngIf="isStatus(requestStatus.Review)">
 
-        <div *ngIf="isReviewStatus(requestReviewStatus.Validation)">
-            <button type="button" class="btn btn-danger" (click)="rejectRequest()">Reject</button>
+    <form>
+        <!-- Draft / Submit -->
+        <div class="action-container" *ngIf="isStatus(requestStatus.Draft)">
+            <button type="button" class="btn btn-outline-danger" id="reset-form-btn" (click)="resetForm()">
+                <span jhiTranslate="request.toolbar.reset">Reset</span>
+            </button>
             <div class="float-xs-right">
-                <button type="button" class="btn btn-default" (click)="requireRevision()">Request revision</button>
-                <button type="submit" class="btn btn-primary"
-                    (click)="validateRequest()">Validate</button>
+                <button type="button" class="btn btn-outline-primary" id="save-draft-btn" (click)="saveDraft()">
+                    <span jhiTranslate="request.toolbar.saveDraft">Save draft</span>
+                </button>
+                <button type="submit" class="btn btn-outline-success" id="submit-draft-btn"
+                        [disabled]="!form.form.valid" (click)="submitDraft()">
+                    <span jhiTranslate="request.toolbar.submit">Submit</span>
+                </button>
             </div>
         </div>
-        <div *ngIf="isReviewStatus(requestReviewStatus.Review)">
-            <button type="button" class="btn btn-danger" (click)="rejectRequest()">Reject</button>
-            <div class="float-xs-right">
-                <button type="button" class="btn btn-default" (click)="assignReview()">Assign review</button>
-                <button type="submit" class="btn btn-primary"
-                    (click)="validateRequest()">Validate</button>
+
+        <!-- Review -->
+        <div class="action-container" *ngIf="isStatus(requestStatus.Review)">
+
+            <!-- Researcher actions -->
+            <div *ngIf="isRequestingResearcher()">
+
+                <!-- Validation -->
+                <div *ngIf="isReviewStatus(requestReviewStatus.Validation)">
```

Comment:
Since it's been used almost all over the place, `isReviewStatus(requestReviewStatus.Validation)` can actually potentially be a directive?